README.md

```diff
@@ -50,7 +50,9 @@ None
 
 This variable is a dict of variables of configuration files, such as
 `/etc/default/saslauthd`. A key in the dict is name of the variable in the
-file, and the value of the key is value of the variable.
+file, and the value of the key is value of the variable. The variable is
+combined with `cyrus_sasl_saslauthd_flags_default`, which includes upstream's
```

Comment:
`include` means "have something as a part of if". to clarify nothing other than the default values are included, "keeps" (or something similar word) is appropriate.